package.json

```diff
@@ -52,7 +52,8 @@
     "README.md"
   ],
   "jest": {
-    "testRegex": "(/tests/.*|(\\.|/)(test|spec))\\.jsx?$"
+    "testRegex": "(/tests/.*|(\\.|/)(test|spec))\\.jsx?$",
+    "testURL": "http://localhost"
```

Comment:
@AnderRV is this line necessary??

Reply:
@benatespina there is an error when running tests without that:
`SecurityError: localStorage is not available for opaque origin`